Source/Core/Core/ConfigManager.cpp

```diff
@@ -325,7 +325,7 @@ void SConfig::SaveNetworkSettings(IniFile& ini)
 
   network->Set("SSLDumpRead", m_SSLDumpRead);
   network->Set("SSLDumpWrite", m_SSLDumpWrite);
-  network->Set("SSLVerifyCert", m_SSLVerifyCert);
+  network->Set("SSLVerifyCertificates", m_SSLVerifyCert);
```

Comment:
That's strange because my portable builds aren't affected (and these versions have this feature, so that's strange). LGTM, then.